src/main/java/io/vlingo/symbio/store/journal/inmemory/InMemoryJournal.java

```diff
@@ -87,6 +88,12 @@ public InMemoryJournal(final Dispatcher<Dispatchable<Entry<T>,RS>> dispatcher, f
     interest.appendResultedIn(Success.of(Result.Success), streamName, streamVersion, source, Optional.empty(), object);
   }
 
+  @Override
+  public <S, ST> void append(String streamName, int streamVersion, Source<S> source, Metadata metadata,
+          AppendResultInterest interest, Object object) {
+    //TODO implement
```

Comment:
Do you have a way to track these unimplemented methods to ensure they are implemented?